tdrs-backend/tdpservice/data_files/util.py

```diff
@@ -4,54 +4,40 @@
 import xlsxwriter
 import calendar
 from tdpservice.parsers.models import ParserErrorCategoryChoices
+from django.conf import settings
+from django.core.paginator import Paginator
 
 
-def get_xls_serialized_file(data):
-    """Return xls file created from the error."""
+def format_error_msg(error_msg, fields_json):
+    """Format error message."""
+    for key, value in fields_json['friendly_name'].items():
+        error_msg = error_msg.replace(key, value) if value else error_msg
+    return error_msg
+
+
+def friendly_names(fields_json):
+    """Return comma separated string of friendly names."""
+    return ','.join([i for i in fields_json['friendly_name'].values()])
```

Comment:
maybe consider adding `if 'friendly_name` in fields_json` or something to that effect, just in case the errors json doesn't have anything in it.

Reply:
never mind, i see that this is done in `check_fields_json` below